### PR TITLE
Add tests for AddSessionCommandParser

### LINKED ISSUE #177: Test- AddSessionCommandParser
Add tests for addSessionCommandParser.
Sessionn =/= Recurring Session
Make consistent with existing parser tests.

### PR DESCRIPTION
**Pull request by JonahhGohh**
@nowknowing thanks for making such detailed test cases for AddRecurringSessionCommandParser!

Resolves #177

### CODE REVIEW

**Review by nowknowing — APPROVED** [review]
LGTM

File: src/test/java/seedu/address/logic/parser/AddSessionCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+        // multiple names - last name accepted
+        assertParseSuccess(parser, NAME_DESC_AMY + NAME_DESC_BOB + SESSION_VALID_DATE + SESSION_VALID_TIME
+                        + SESSION_VALID_DURATION + SESSION_VALID_SUBJECT + SESSION_VALID_FEE,
+                new AddSessionCommand(session, new Name(VALID_NAME_BOB)));
+
+        // multiple session date - last accepted
+        assertParseSuccess(parser, PREAMBLE_WHITESPACE + NAME_DESC_AMY + SESSION_VALID_DATE + SESSION_VALID_DATE
+                        + SESSION_VALID_TIME + SESSION_VALID_DURATION + SESSION_VALID_SUBJECT + SESSION_VALID_FEE,
+                new AddSessionCommand(session, new Name(VALID_NAME_AMY)));
```
Comment (samleewy) [inline src/test/java/seedu/address/logic/parser/AddSessionCommandParserTest.java]: If we're checking for only last session date, maybe it'll be better to use another date so as to differentiate whether it's the first or last session date? seems to be the same for both now.

**Review by samleewy — COMMENTED** [review]
some comments!

File: src/test/java/seedu/address/logic/parser/AddSessionCommandParserTest.java (hunk 1 above)
Comment (JonahhGohh) [reply src/test/java/seedu/address/logic/parser/AddSessionCommandParserTest.java]: Good point!

**Pull request merged**